JPATask/src/main/java/com/instinctools/egor/mentoring/JPA/DAO/ClientDAO.java

```diff
@@ -0,0 +1,17 @@
+package com.instinctools.egor.mentoring.JPA.DAO;
+
+import com.instinctools.egor.mentoring.JPA.Entity.Account;
+import com.instinctools.egor.mentoring.JPA.Entity.Client;
+
+import java.sql.SQLException;
+import java.util.List;
+
+public interface ClientDAO {
+    public void createClient(Client client) throws SQLException;
+    public void deleteClient(Client client) throws SQLException;
+    public List<Client> getAllClients() throws SQLException;
+    public void createAccount(Client client, Account account) throws SQLException;
```

Comment:
I would suggest to not add any **Acount methods here. You can do same by:
1) Adding 'updateClient' method
2) All account management staff can be moved to Client entity (accounts is just a collection - so just iterate with this collection no need to iterates with "ACCOUNT" DB table directly, lets hibernate do your work)